tests/components/group/test_cover.py

```diff
@@ -454,3 +484,27 @@ async def test_set_tilt_positions(hass, setup_comp):
     assert state.attributes[ATTR_CURRENT_TILT_POSITION] == 80
 
     assert hass.states.get(DEMO_COVER_TILT).attributes[ATTR_CURRENT_TILT_POSITION] == 80
+
+
+@pytest.mark.parametrize("config_count", [(CONFIG_POS, 2)])
+async def test_is_opening(hass, setup_comp):
+    """Test is_opening property."""
+    await hass.services.async_call(
+        DOMAIN, SERVICE_OPEN_COVER, {ATTR_ENTITY_ID: COVER_GROUP}, blocking=True
+    )
+
+    assert hass.states.get(COVER_GROUP).state == STATE_OPENING
```

Comment:
We should also check the other way around, if we change a group member state, does the group state change.

Reply:
I think that's redundant. The cover group entity gets it's state from the group members in `async_update` (and only there). If the group entity changes it's properties it is because the group members already have changed.
That the group members indeed change their state is tested below.
(And the change of the group members due to the open/close service call is tested in `test_open_covers`/`test_close_covers`.

Reply:
I would like to see this kind of test but for opening and closing states:
https://github.com/home-assistant/core/blob/acfd907c50fc54a4aad7f1166a98fd6ddb46ef84/tests/components/group/test_cover.py#L85-L89